articles/support/delete-reset-tenant.md

```diff
@@ -16,6 +15,12 @@ useCase:
 
 Using the [Reset Tenant Extension](https://github.com/auth0-extensions/auth0-reset-tenant), you can programmatically make changes to your Auth0 tenant. Not only is this useful for creating new environments for development or test purposes, you can use the extension to "clean" your tenant and remove unwanted items.
 
+::: note
```

Comment:
This title is about delete vs reset tenant. I'd put this note under the "Delete your Auth0 Tenant" title instead, after the warning that deletion cannot be restored.

Reply:
Also for that warning, instead of "This action cannot be undone." I'd say "Deleted tenants cannot be restored and the tenant name may not be used again when creating new tenants." Same as in here https://auth0.com/docs/policies/restore-deleted-tenant.

Reply:
I'm rewriting the entire article to move notes and warnings to appropriate sections.